src/resolvers/queryNodeStitchingResolvers/videoResolvers.ts

```diff
@@ -9,16 +9,30 @@ import { OrionContext } from '../../types'
 export const videoResolvers = (queryNodeSchema: GraphQLSchema): IResolvers<any, OrionContext> => ({
   Query: {
     top10VideosThisWeek: async (parent, args, context, info) => {
-      const mostViewedVideosIds = getMostViewedVideosIds(context, { limit: 10, period: 7 })
+      const mostViewedVideosIds = getMostViewedVideosIds(context, { limit: 30, period: 7 })
       const resolver = createResolver(queryNodeSchema, 'videos')
-      const videos = await getDataWithIds(resolver, mostViewedVideosIds, parent, args, context, info)
-      return sortEntities(videos, mostViewedVideosIds)
+      const videos = await getDataWithIds(
+        resolver,
+        mostViewedVideosIds,
+        parent,
+        { ...args, isCensored_eq: false, isPublic_eq: true },
```

Comment:
We already pass those from Atlas and we do it this way for few other queries as well, I think it's better to stay consistent and leave it as it was